### PR TITLE
Exercise endpoints

### PR DESCRIPTION
**Pull request by Chulstro**
# SOM-Timer BE Pull Request Form

## Issues Addressed by this Pull Request
 HOTFIX FOR CORS

### CODE REVIEW

**Review by rwilliams659 — APPROVED** [review]
Thanks for helping us with the CORS issue!

**Review by jkwest-93 — APPROVED** [review]
Thanks so much for being able to work on this so quickly!!

File: application/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from flask import Flask, jsonify
+from flask_cors import CORS, cross_origin
```
Comment (sienna-kopf) [inline application/__init__.py:2]: So interesting. I've never seen this before

File: requirements.txt [hunk 2]
```diff
@@ -10,3 +10,4 @@ six
 flask-testing
 coverage
 gunicorn
+flask_cors
```
Comment (sienna-kopf) [inline requirements.txt:13]: I hope this works! Can't wait to research this more later!

**Review by sienna-kopf — APPROVED** [review]

**Pull request merged**